Resources/Audio/Items/attributions.yml

```diff
@@ -27,3 +27,8 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken from tgstation, modified by brainfood1183 (github) for ss14"
   source: "https://github.com/tgstation/tgstation/blob/529d97cb1c105bcd548e95a9c9070bbf5253dd81/sound/items/AirHorn.ogg"
+
+- files: ["desk_bell_ring.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Taken from tgstation"
+  source: "https://github.com/tgstation/tgstation/blob/master/sound/machines/microwave/microwave-end.ogg"
```

Comment:
same
Also this sound was added to to TG in this PR https://github.com/tgstation/tgstation/pull/32500 the author seems to still be active so please reach out to them and ask them if 1) They **made** that sound 2) They agree with this license . If both are true you need to make the copyright "Made by *author name* for tgstation"

Reply:
Okay, done (x2)

Reply:
well, what did they say? can you post an screenshot or get them to comment here